test/test.lua

```diff
@@ -34,6 +34,16 @@ local function isEqual(a, b, tolerance, ...)
    end
 end
 
+local function checkMultiDevice(x, fn, ...)
```

Comment:
I see. Didn't realize that the tensor is created on a different device than the one we run the kernel on.